gpio/python/gpioPython/.vscode/settings.json

```diff
@@ -1,18 +1,22 @@
 {
-  "torizon_psswd": "",
-  "torizon_login": "",
-  "torizon_ip": "",
-  "host_ip": "",
+  "torizon_psswd": "1",
+  "torizon_login": "torizon",
+  "torizon_ip": "10.22.1.63",
```

Comment:
"Remove Default" on the device before commiting, so it doesn't add your devices configuration